wiki/People/Beatmap_Nominators/Rules/en.md

```diff
@@ -40,12 +40,12 @@ Beatmap Nominators possess the tools to nominate beatmaps and shape the world of
 + **Disqualifications should primarily be done for beatmaps with unrankable issues, or if the beatmapset owner requests it.**
 + **Disqualification requests by the beatmapset owner can be applied immediately.** This includes yourself, in case you want to self-dq.
 + **Issues that can be mitigated through online offsets, tags or similar must still be disqualified.** In the case that this is not possible, contact a member of the NAT. Online offsets and tags only apply for the website, so players will still have the issues in the client unless fixed before ranked.
-+ **Qualified beatmaps found with unrankable issues must first be posted to the [Report a Beatmap](https://osu.ppy.sh/community/forums/topics/447428) thread.** If the issue has already been reported by someone else, then you are free to disqualify the beatmap. If there is less than 48 hours until rank, it may be disqualified immediately. Getting a second person to confirm greatly reduces the chance that disqualifications are mistakes and is often quick to do.
-+ **For non-unrankable issues or concerns, the mapper should be given time until 24 hours before estimated rank time to respond.** If the mapper fails to respond, the beatmap may be disqualified for further discussion. This does not apply to posts made less than 48 hours before the estimated rank date of the beatmapset. This is to give mappers ample time to reply and decide whether to make changes. 
++ **Qualified beatmaps found with unrankable or significant issues must be disqualified immediately.**
++ **Do not disqualify beatmaps of game modes you are not formally qualified for.** This does not apply to disqualifications regarding audio, backgrounds, and other content that does not directly pertain to the beatmapset's game mode.
```

Comment:
```suggestion
+ **Do not disqualify beatmaps of game modes you are not formally qualified for.** This does not apply to disqualifications regarding audio, backgrounds, and other content that does not directly pertain to the beatmaps' game modes.
```
beatmapsets don't have game modes /shrug

Reply:
This needs to be changed in the veto section as well then will do soon